source/image-handler/test/image-handler/crop.spec.ts

```diff
@@ -10,16 +10,21 @@ import { ImageEdits, StatusCodes } from "../../lib";
 const s3Client = new S3();
 const rekognitionClient = new Rekognition();
 
+// base64 encoded images
+const image_png_1x1 =
```

Comment:
do wee need this extra image? It seems to only be used in the crop out of bounds test. Could we instead re-use the image_png_white_1x1 item?

Reply:
The extra image was needed because it had to have a white background to match the image_png_white_5x5 image background.
The image_png_1x1 image could not be scaled up to 5x5 so so 2 new images were used.

Reply:
Couldn't you use image_png_white_1x1 in place of image_png_1x1, or likewise use 5x5 version of image_png_1x1 and avoid the extra variable?

Reply:
Changed to re-use image_png_white_1x1 and avoid an additional image